apps/emqx/src/emqx_schema.erl

```diff
@@ -858,6 +858,14 @@ fields("mqtt_quic_listener") ->
                 ?MAX_UINT(16),
                 ?DESC(fields_mqtt_quic_listener_stateless_operation_expiration_ms)
             )},
+        {"sslkeylogfile",
```

Comment:
Nit: wouldn't this fit better inside `ssl_options`?

Reply:
This is QUIC (quicer) specific and cannot be reused by TLS transport yet.